data/LRWholeGenomeSingleSample/TestWithONT.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This script promotes me to step back and look at the current organization of files.
They look like the following
```bash
$ tree -d
.
├── data
│   ├── CanuTest
│   ├── DownloadFromSRA
│   ├── FindMTPrimersWorkflow
│   ├── LRWholeGenomeSingleSample
│   ├── MHCAsmWorkflow
│   ├── PBMultiSampleWorkflow
│   └── PBSingleSampleWorkflow
├── docker
│   ├── lr-align
│   ├── lr-asm
│   └── lr-cloud-downloader
├── reports
│   ├── Genentech
│   └── OngoingReport
├── resources
│   └── workflow_options
├── scripts
└── wdl
    └── old

20 directories
```

I'm thinking about re-organizing the directories in this way.
```bash
$ tree -d
.
├── data
│   └── various_types_of_data...
├── reports
│   ├── Genentech
│   └── OngoingReport
└── workflow
    ├── docker
    ├── json
    │   ├── options
    │   └── test_inputs
    └── wdl

11 directories
```

What do you think?

Reply:
This is worth some debate.  I'm trying to think about it from the perspective of someone outside the institute trying to run these pipelines.

I can see why moving the docker/ directory under workflow/ would make sense in that scenario - the docker images and the workflows are linked and should travel together.

The workflow options are not *actually* part of the workflows themselves; anyone could swap in their own options and be on their way.  The workflow inputs are *definitely* not part of the workflows.

Right now the repo is both where we're writing pipelines *and* running pipelines.  But eventually, once the development here matures, we should really strip out the input JSONs (and everything else not really meant to be redistributed to the world) keep them somewhere else.

Reply:
Agree. 
Then the `dockers` and `wdl` directories should be moved into a new `workflow`, right?